### PR TITLE
Staking app

### PR DESCRIPTION
**Pull request by Pinkdesu**
- Added the staking app

### CODE REVIEW

**Review by grishasobol — APPROVED** [review]
Norm

File: apps/staking/src/components/layout/header/account/accounts/Accounts.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import type { InjectedAccountWithMeta } from '@polkadot/extension-inject/types';
+import { useAccount } from '@gear-js/react-hooks';
+import { isLoggedIn } from 'utils';
+import { LOCAL_STORAGE } from 'consts';
+import { AccountButton } from '../account-button';
+import styles from './Accounts.module.scss';
```
Comment (aderrod94) [inline apps/staking/src/components/layout/header/account/accounts/Accounts.tsx:6]:
```
import type { InjectedAccountWithMeta } from '@polkadot/extension-inject/types';
import { useAccount } from '@gear-js/react-hooks';

import styles from './Accounts.module.scss';
import { AccountButton } from '../account-button';

import { isLoggedIn } from 'utils';
import { LOCAL_STORAGE } from 'consts';
```

File: apps/staking/src/components/common/switcher/Switcher.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  items: SwitchItem[];
+  onChange: (value: string) => void;
+};
+
+function Switcher({ value: currentValue, items, onChange }: Props) {
+  const getClasses = (value: string) => clsx(styles.switchBtn, value === currentValue && styles.active);
+
+  return (
+    <ul className={clsx(styles.switcher, 'customScroll')}>
+      {items.map(({ value, label }) => (
```
Comment (aderrod94) [inline apps/staking/src/components/common/switcher/Switcher.tsx:17]: I think better to use function that return `li`

File: apps/staking/src/components/common/switcher/Switcher.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <li key={value}>
+          <button type="button" className={getClasses(value)} onClick={() => onChange(value)}>
```
Comment (aderrod94) [inline apps/staking/src/components/common/switcher/Switcher.tsx:19]: Also think better to use handleClick

File: apps/staking/src/components/layout/header/account/wallet/Wallet.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { Account } from '@gear-js/react-hooks';
+import { AccountButton } from '../account-button';
+import styles from './Wallet.module.scss';
```
Comment (aderrod94) [inline apps/staking/src/components/layout/header/account/wallet/Wallet.tsx:3]:
```
import { Account } from '@gear-js/react-hooks';

import styles from './Wallet.module.scss';
import { AccountButton } from '../account-button';

```

File: apps/staking/src/components/layout/header/logo/Logo.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { Link } from 'react-router-dom';
+import { ReactComponent as SVG } from 'assets/images/logo.svg';
```
Comment (aderrod94) [inline apps/staking/src/components/layout/header/logo/Logo.tsx:2]:
```
import { Link } from 'react-router-dom';

import { ReactComponent as SVG } from 'assets/images/logo.svg';
```

File: apps/staking/src/components/loaders/api-loader/index.ts [hunk 6]
```diff
@@ -0,0 +1,3 @@
+import { ApiLoader } from './ApiLoader';
```
Comment (aderrod94) [inline apps/staking/src/components/loaders/api-loader/index.ts:1]: `export { ApiLoader } from './ApiLoader';`

File: apps/staking/src/hocs/index.tsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import {
+  ApiProvider as GearApiProvider,
+  AlertProvider as GearAlertProvider,
+  AccountProvider,
+  ProviderProps,
+} from '@gear-js/react-hooks';
+import { Alert, alertStyles } from '@gear-js/ui';
+import { ComponentType } from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { ADDRESS } from 'consts';
```
Comment (aderrod94) [inline apps/staking/src/hocs/index.tsx:10]:
```
import {
  ApiProvider as GearApiProvider,
  AlertProvider as GearAlertProvider,
  AccountProvider,
  ProviderProps,
} from '@gear-js/react-hooks';
import { Alert, alertStyles } from '@gear-js/ui';
import { ComponentType } from 'react';
import { BrowserRouter } from 'react-router-dom';

import { ADDRESS } from 'consts';
```

File: apps/staking/src/components/layout/footer/socials/Socials.tsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
+import { ReactComponent as Twitter } from 'assets/images/socials/twitter.svg';
+import { ReactComponent as Github } from 'assets/images/socials/github.svg';
+import { ReactComponent as Discord } from 'assets/images/socials/discord.svg';
+import { ReactComponent as Medium } from 'assets/images/socials/medium.svg';
+import styles from './Socials.module.scss';
```
Comment (aderrod94) [inline apps/staking/src/components/layout/footer/socials/Socials.tsx:5]:
```
import styles from './Socials.module.scss';

import { ReactComponent as Twitter } from 'assets/images/socials/twitter.svg';
import { ReactComponent as Github } from 'assets/images/socials/github.svg';
import { ReactComponent as Discord } from 'assets/images/socials/discord.svg';
import { ReactComponent as Medium } from 'assets/images/socials/medium.svg';
```

**Review by aderrod94 — APPROVED** [review]

**Pull request merged**